### DISCUSSION TITLE
是否会加入i18n解决方案

### DISCUSSION THREAD

**Discussion by oNo500** [weapp-vite/weapp-vite#50] [Ideas]
比如加入对官方[miniprogram-i18n](https://github.com/wechat-miniprogram/miniprogram-i18n) 的支持？目前微信生态似乎没有一个最佳的解决方案。

**Comment by sonofmagic** [comment]
目前会按照小程序里面生态最好的方案去支持， 目前感觉各个方案都一般般吧

**Reply by sonofmagic** [reply]
我感觉可以原生支持一下 @miniprogram-i18n/core
